Javascript Course: feat: added referenced lessons url for easy navigation (#29321)
TheOdinProject/curriculum: javascript/asynchronous_javascript_and_apis/async_and_await.md

### Practice

> Remember the Giphy API [practice project](https://www.theodinproject.com/lessons/node-path-javascript-working-with-apis#lets-do-this)? (If not, you should go back and complete the [API lesson](https://www.theodinproject.com/lessons/node-path-javascript-working-with-apis)). We are going to convert the promise based code into `async/await` compatible code. Here's a refresher of the code we are starting with:

Reviewer: ```suggestion
Remember the [Giphy API practice project](https://www.theodinproject.com/lessons/node-path-javascript-working-with-apis#lets-do-this) from the previous lesson? We are going to convert the promise based code into `async/await` compatible code. Here's a refresher of the code we are starting with:
```

Just a suggestion to further enhance the change:
- "practice project" isn't accessible enough of a link text label but "Giphy API practice project" would be.
- We don't really need to link to both the Giphy section *and* the API lesson right next to each other, a bit redundant. The Giphy section is more relevant and the API lesson could just be referred to as the "previous lesson" which IMO flows a bit better anyway.
- Shouldn't need to say "if not, you should go back" - the curriculum expects to be done in order anyway so people shouldn't be skipping to this lesson without having done the API one. In other lessons where there was this sort of language, those parts were removed because it just didn't make sense to accommodate for potential skipping.

Author: I won't be able  to contribute further at the moment. I’m currently occupied with exams for the next couple of months, so now that this has been reviewed, anyone else is welcome to take over the pr...
